### PR TITLE
Add DARIAH-DE Geo-Browser to tools

### PR DESCRIPTION
**Pull request by fungunga**
Let's try how it'll work! :-) Thanks!

### CODE REVIEW

**Comment by sbudden** [comment]
@emanueldima Let's see what Emanuel thinks of it.

File: tools/DARIAH-DE-Geo-Browser-CSV.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "task": "Visualisation",
+    "deployment": "development",
+    "name": "DARIAH-DE Geo-Browser (CSV)",
+    "logo": "dariah.svg",
```
Comment (emanueldima) [inline tools/DARIAH-DE-Geo-Browser-CSV.json:5]: The file name does not match the .svg (it's `dariah-de` and not `dariah`)

File: tools/DARIAH-DE-Geo-Browser-KML.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "task": "Visualisation",
+    "deployment": "development",
+    "name": "DARIAH-DE Geo-Browser (KML)",
+    "logo": "dariah.svg",
```
Comment (emanueldima) [inline tools/DARIAH-DE-Geo-Browser-KML.json:5]: Same naming error as in the other json file

**Comment by emanueldima** [comment]
I added the tools to my development version of the switchboard, https://weblicht.sfs.uni-tuebingen.de/switchboard-test/, but I don't have a good test file. Could you provide a few sample files of different types?

File: tools/DARIAH-DE-Geo-Browser-CSV.json (hunk 1 above)
Comment (fungunga) [reply tools/DARIAH-DE-Geo-Browser-CSV.json]: thanks, changed!

File: tools/DARIAH-DE-Geo-Browser-KML.json (hunk 2 above)
Comment (fungunga) [reply tools/DARIAH-DE-Geo-Browser-KML.json]: Thanks again :-) Also fixed!

**Comment by fungunga** [comment]
(application/vnd.dariahde.geobrowser.csv):
https://trep.de.dariah.eu/1.0/dhcrud/21.T11991/0000-001A-F5EF-3
link-to-gb:
https://geobrowser.de.dariah.eu/beta?csv=https://trep.de.dariah.eu/1.0/dhcrud/21.T11991/0000-001A-F5EF-3

(text/csv):
https://trep.de.dariah.eu/1.0/dhcrud/21.T11991/0000-001A-F5F3-D
link-to-gb:
https://geobrowser.de.dariah.eu/beta?csv=https://trep.de.dariah.eu/1.0/dhcrud/21.T11991/0000-001A-F5F3-D

(application/vnd.google-earth.kml+xml):
https://geobrowser.de.dariah.eu/data/kafka.kml
link-to-gb:
https://geobrowser.de.dariah.eu/beta/?kml=https://geobrowser.de.dariah.eu/data/kafka.kml

**Comment by emanueldima** [comment]
Thanks for the example files. I'm not exactly sure why, but the geobrowser does not show any marker on the map when invoked from the switchboard. Could you try and debug it? You have to follow these steps:

1. Go to https://weblicht.sfs.uni-tuebingen.de/switchboard-test/
2. Upload the kml file (kafka.kml)
3. The Switchboard will recognize its mediatype, but not the language. Select "English" as the language.
4. The GeoBrowser will be matched and will show up in the list of matched tools. Click the "Open" green button which will send you to the geobrowser

The Switchboard sends the query parameters encoded (application/x-www-form-urlencoded) but in your working examples the parameters are not. This could be one of the problems.

**Comment by fungunga** [comment]
https://geobrowser.de.dariah.eu/beta/?csv=https%3A%2F%2Fweblicht.sfs.uni-tuebingen.de%2Fswitchboard-test%2Fapi%2Fstorage%2F1311ec19-c810-4fa4-9557-ca399c61e4f8
is working (CSV), and
https://geobrowser.de.dariah.eu/beta/?kml=https%3A%2F%2Fweblicht.sfs.uni-tuebingen.de%2Fswitchboard-test%2Fapi%2Fstorage%2F51fe3161-7fec-4e53-96b8-5838492ed92b
is also working (now) (KML)…
https://weblicht.sfs.uni-tuebingen.de/switchboard-test/#/trep/https%3A%2F%2Ftrep.de.dariah.eu%2F1.0%2Fdhcrud%2Fhdl:21.T11991%2F0000-001A-F5F3-D%2Fdata/text%2Fcsv
and
https://weblicht.sfs.uni-tuebingen.de/switchboard-test/#/trep/https%3A%2F%2Ftrep.de.dariah.eu%2F1.0%2Fdhcrud%2Fhdl:21.T11991%2F0000-001A-F5EF-3%2Fdata/application%2Fvnd.dariahde.geobrowser.csv
does not set the given mimetype?

BTW Can we set the language via API? Or do something about it in the services' documentation?

**Comment by fungunga** [comment]
Would it make sense to use text/plain also for the Geo-Browser?

**Comment by emanueldima** [comment]
Thanks, that works. I'd say to not use text/plain for now, it would be misleading.

**Pull request merged**